packages/embed-react/README.md

```diff
@@ -261,6 +262,40 @@ embed and instead you will need to handle the form submission. You should implem
 />
 ```
 
+## Updating transaction metadata and external identifier
+
+By default Embed will use the options you pass when initializing Embed for every transaction request. These options can be dynamically changed for every transaction request using the `onBeforeTransaction` callback. This is useful in case you want to
```

Comment:
Is this right for every transaction request?

I dont think we generate a new idempotency key do we.

Reply:
Yes - every transaction request can generate new options. @cbetta Correct me if I'm wrong - but the responsibility is on the merchant not to change these value if they want idempotency?

Reply:
Yes this is right.

Based on the conversation yesterday this was also fine.

* If a TX fails repeatedly (5xx/no response/409) then we show an error screen and prevent resubmission
* If a TX fails for other reasons the user can retry with a new card/other payment option and if the merchant were to create new options with different values then idempotency wont work, which is fine cause it's a different request.